doc/user_guide.adoc

```diff
@@ -488,6 +488,6 @@ _usage example_: `./gradlew suggestMergedModuleInfo --language=kotlin`
         with --app-image=<font color="saddlebrown">imageOutputDir</font>/<font color="midnightblue">imageName</font>
 ----
 The properties <font color="saddlebrown">imageOutputDir</font>, <font color="cadetblue">installerOutputDir</font>, and <font color="midnightblue">imageName</font>
```

Comment:
These colors are not being rendered correctly, but I did not know how to fix that without changing the colors to one of the [HTML color names](https://en.wikipedia.org/wiki/Web_colors#HTML_color_names).